tap_tiktok_ads/client.py

```diff
@@ -7,15 +7,24 @@
 
 LOGGER = singer.get_logger()
 
+ENDPOINT_BASE = "https://{api}.tiktok.com/open_api/v1.2"
+# SANDBOX_ENDPOINT_BASE = "https://sandbox-ads.tiktok.com/open_api/v1.2"
```

Comment:
Remove commented lines, An why this PR raised to [TDL-17937-dict-based-to-class-refactoring](https://github.com/singer-io/tap-tiktok-ads/tree/TDL-17937-dict-based-to-class-refactoring) branch?

Reply:
- Removed commented lines
- Because we wanted that code and at the time of merging it gives conflicts, so all the branches have the dict to class based code

Reply:
Got it, but do add this information in the PR description and card. Approving the PR.

Reply:
Added description